src/state/items.ts

```diff
@@ -46,21 +47,28 @@ export const isUserCanBuyAnItem = async (
 export const listAThing = async (
   mintbase: IWallet,
   metadata: ItemWithMetadata
-) => {
+): Promise<boolean> => {
   let tokenId = metadata.tokens[0].id
   tokenId = tokenId.split(':')[0]
-  await mintbase.list(tokenId, mintbaseContract, defaultPrice, {
-    autotransfer: true,
-  })
+  const { data } = await mintbase.list(
```

Comment:
list method can return data or error. It should be `const { data, error } ....` and throw an error if any